CONTRIBUTING.md

```diff
@@ -149,6 +159,42 @@ brew install cmake qt5
 brew link qt5 --force
 ```
 
+### Android
+
+To build for Android, you must download components of the Android SDK, the Android NDK, and Java Development Kit.
+
+If you've already got the tools required, simply set the following three environment variables:
+```
+export ANDROID_SDK=<path_to_sdk_root>
+export ANDROID_NDK=<path_to_ndk_root>
+export JAVA_HOME=<path_to_jdk_root>
+```
+Otherwise, below are steps to acquire the tools for Ubuntu.
+
+The Java Development Kit can be installed with:
+```
+sudo apt-get install openjdk-8-jdk
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+```
+
+The Android SDK and NDK can be set up with the following steps.  They are also mirrored in our Travis-CI [setup script](scripts/travis/android_setup.sh) for Android.  We are currently targeting build-tools 26.0.1 and NDK r14b.
+```
+# Set up Android SDK and NDK
+export ANDROID_SDK=<path_to_desired_setup>
+pushd $ANDROID_SDK
+wget http://dl.google.com/android/repository/sdk-tools-linux-3859397.zip
+unzip sdk-tools-linux-3859397.zip
+cd tools/bin/
+./sdkmanager --sdk_root=$ANDROID_SDK "build-tools;26.0.1" "platforms;android-23"
+# Accept the license
+
+# Set up Android NDK
+pushd $ANDROID_SDK
+wget http://dl.google.com/android/repository/android-ndk-r14b-linux-x86_64.zip
+unzip android-ndk-r14b-linux-x86_64.zip
+export ANDROID_NDK=$ANDROID_SDK/android-ndk-r14b
+```
+
```

Comment:
Can you also add instructions for fetching the dependencies on windows here? I think it should be roughly similar to the linux steps (and it's fine to just give wget/unzip commands for the relevant bits where possible) but since it's extra fiddly on windows it would be useful to have it listed here for reference.